mmv1/third_party/terraform/website/docs/d/secret_manager_secrets.html.markdown

```diff
@@ -0,0 +1,132 @@
+---
+subcategory: "Secret Manager"
+description: |-
+  List the Secret Manager Secrets.
+---
+
+# google\_secret\_manager\_secrets
+
+Use this data source to list the Secret Manager Secrets
+
+## Example Usage 
+
+
+```hcl
+data "google_secret_manager_secrets" "secrets" {
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (optional) The ID of the project.
+
+* `filter` - (optional) A filter expression that
+    filters resources listed in the response. The expression must specify
+    the field name, an operator, and the value that you want to use for
+    filtering. The value must be a string, a number, or a boolean. The
+    operator must be either "=", "!=", ">", "<", "<=", ">=" or ":". For
+    example, if you are filtering Compute Engine instances, you can
+    exclude instances named "example-instance" by specifying "name !=
+    example-instance". The ":" operator can be used with string fields to
+    match substrings. For non-string fields it is equivalent to the "="
+    operator. The ":*" comparison can be used to test whether a key has
+    been defined. For example, to find all objects with "owner" label
+    use: """ labels.owner:* """ You can also filter nested fields. For
```

Comment:
Is this triple quote formatting required? I'm not seeing an explanation for it in the description

Reply:
I've updated the field description to be consistent with the [API docs](https://cloud.google.com/secret-manager/docs/reference/rest/v1/projects.secrets/list#query-parameters)